src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Application/Commands/CreateCohort/CreateCohortHandler.cs

```diff
@@ -30,10 +31,25 @@ public async Task<CreateCohortResponse> Handle(CreateCohortRequest request, Canc
             var apiRequest = await _mapper.Map(request).ConfigureAwait(false);
             var apiResult = await _apiClient.CreateCohort(apiRequest, cancellationToken).ConfigureAwait(false);
 
+            var apprenticeships = await _apiClient.GetDraftApprenticeships(apiResult.CohortId, cancellationToken);
+
+            long? hasStandardOptions = null;
```

Comment:
should this be re-named `draftApprenticeshipId`

Reply:
👍🏻 all renamed 🕺🏻